src/pages/en/core-concepts/astro-pages.md

```diff
@@ -20,10 +20,12 @@ Astro supports the following file types in the `src/pages/` directory:
 
 Astro leverages a routing strategy called **file-based routing**. Each file in your `src/pages/` directory becomes an endpoint on your site based on its file path.
 
-Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your component template to link between pages.
-
 📚 Read more about [Routing in Astro](/en/core-concepts/routing/).
 
+### Link between pages
+
+Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your Astro components to link to other pages on your site.
```

Comment:
Should this be “pages”? You’d also use `<a>` in e.g. a React component and these are the page docs after all.

```suggestion
Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your Astro pages to link to other pages on your site.
```

Reply:
Ooh, good call. I was thinking about that...